src/constants.js

```diff
@@ -33,6 +33,13 @@ export const GOAL_STATUS = {
   CLOSED: 'Closed',
 };
 
+export const GRANT_PERSONNEL_ROLES = [
```

Comment:
do these need to be added to `@ttahub/common` as well, or will they only ever be used by this update job?

Reply:
I think its fine to leave these here for now as they are only used by the BE job.